index.md

```diff
@@ -9,55 +9,192 @@ iOS and Android from a single codebase.
 
 {% include note.html content="_Flutter is an early-stage open-source project_. We hope you try Flutter out and send us [feedback](mailto:flutter-dev@googlegroups.com)." %}
 
-## New to Flutter
+## What is Flutter?
 
-See these topics to quickly start building cross-platform mobile apps with Flutter.
+Flutter provides an easy and productive way to build and deploy high-performance 
+mobile apps for Android and iOS. The goal is to deliver apps that feel natural 
+on different platforms, and we embrace differences like scrolling behavior, 
+overscroll behaviors, and more. 
 
- - [Set up your machine](/setup/) with Flutter and iOS/Android tools.
- - [Get started](getting-started) to run your first app on Android and iOS.
- - [A tour of the Flutter framework](widgets-intro)
- - [Bootstrap into Dart](/bootstrap-into-dart/) if you are new to the language
- for developing Flutter apps.
+![Shrine, iOS left and Android right](/images/whatisflutter/shrine_both.png)
 
-## Developer Guides
+This is an online marketplace app from the [Gallery](https://github.com/flutter/flutter/tree/master/examples/flutter_gallery/lib/demo), 
+a showcase of Flutter apps to download and run on a phone. Shrine has 
+high-quality scrolling images, interactive cards, buttons and dropdown lists, 
+and a shopping cart page. Run Shrine on an iPhone or Android phone; the UI 
+appearance and behavior are equivalent. Feel free to browse through the source 
+code and other sample apps [here](https://github.com/flutter/flutter/tree/master/examples) 
+in our github repository.
+
+No mobile development experience is required to get started. Apps are written 
+in [Dart](https://dartlang.org), which will look familiar if you've used a 
+language like Java or JavaScript. While experience with object-oriented languages 
+is definitely helpful, even non-programmers have made Flutter apps!
 
-See these topics to understand how the Flutter framework works.
+## Why use Flutter?
 
- - [Widgets overview](widgets)
- - [Gestures in Flutter](gestures)
- - [Animations in Flutter](animations)
- - [Using custom fonts](custom-fonts)
- - [Layered design (video)](https://www.youtube.com/watch?v=dkyY9WCGMi0)
- - [Rendering pipeline (video)](https://www.youtube.com/watch?v=UUfXWzp0-DU)
- - [Dealing with box constraints in Flutter](layout)
+What are some advantages of Flutter? It helps you:
```

Comment:
Will definitely keep an eye out for ways to work in these selling points, and justify them so that users don't question whether we can fairly make the claims. This was recent UX feedback we got - on "high-performance". It was hard work to string together a stripped-down narrative to show what we meant, why it was fact-based and not just marketing.

Flutter is great, and it does these things, and let's work at demonstrating that.